scripts/globals/items/demonic_sword.lua

```diff
@@ -31,6 +32,6 @@ function onAdditionalEffect(player,target,damage)
             message = 167;
         end
 
-        return SUBEFFECT_DARKNESS_DAMAGE,message,dmg;
+        return SUBEFFECT_DARK_DAMAGE,message,dmg;
```

Comment:
SUBEFFECT_DARKNESS_DAMAGE was correct, SUBEFFECT_DARK_DAMAGE does not exist in status.lua